packages/devextreme/js/__internal/grids/grid_core/keyboard_navigation/m_keyboard_navigation.ts

```diff
@@ -2841,8 +2841,11 @@ const editing = (Base: ModuleType<EditingController>) => class EditingController
     keyboardNavigation._fastEditingStarted = false;
 
     const result = super.closeEditCell.apply(this, arguments as any);
+    const [,,isFilterCell] = arguments;
```

Comment:
Let's restore the normal usage of the args:
`public closeEditCell(isError?, withoutSaveEditData?, isFilterCell?) {`
`const result = super(isError, withoutSaveEditData, isFilterCell);`
`const [,,isFilterCell] = arguments; // just remove`